landscaper/gardenlet/README.md

```diff
@@ -0,0 +1,125 @@
+Essentially follows this [documentation](../../docs/deployment/deploy_gardenlet_manually.md).
+
+## Prerequisites
+
+The gardenlet landscaper deploys a `VerticalPodAutoscaler` resource
+**if VPA for the Gardenlet is enabled** (via  field `.deploymentConfiguration.vpa` in the [import configuration](#import-configuration).
+Hence, in this case the Kubernetes cluster to be registered as a Seed must have 
+[Vertical Pod Autoscaling](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) enabled 
+(requires at least the CRD `VerticalPodAutoscaler` to be registered)
+
+## Run
+
+The Gardenlet landscaper component is supposed to run as part of the [Gardener Landscaper](https://github.com/gardener/landscaper) 
+but can also be executed stand-alone.
+
+The Gardenlet landscaper only supports one command line argument 
+ - `--integration-test true/false` (defaults to `false`).
+   
+   In integration tests, we do not assume that the Gardenlet can be rolled out successfully,
+   nor that the Seed can be registered.
+   This is to provide an easy means of testing the landscaper component without requiring
+   a fully functional Gardener control plane.
+
+
+Make sure to set the following environment variables:
+- `IMPORTS_PATH` contains the path to a file containing [required import configuration](#required-configuration).
+- `OPERATION` is set to either `RECONCILE` (deploys the Gardenlet) or `DELETE` to remove the deployed resources.
+- `COMPONENT_DESCRIPTOR_PATH` contains the path to a file containing a component descriptor for the Gardenlet. 
+   This file contains the OCI image reference to use for the Gardenlet deployment.
+   You can find a sample descriptor [here](example/run-locally/gardenlet-landscaper-component-descriptor-list.yaml).
+  
+
+### Run locally
+
+1. Run `make dev-setup` to create a default component descriptor and import file for local execution 
+in the directory `dev/landscaper`.
+
+2. Adjust the import file `landscaper/gardenlet-landscaper-imports.yaml` to your setup.
+Please check what can be configured [below](#import-configuration).
+
+3. Finally, run the below `make` statement to run the Gardenlet landscaper. 
+   This already sets the required environment variables and the path to the local imports and component descriptor file.
+
+Run the landscaper gardenlet `RECONCILE` operation:
+
+```
+make start-landscaper-gardenlet-reconcile
+```
+
+Run the landscaper gardenlet `DELETE` operation:
+```
+make start-landscaper-gardenlet-delete
+```
+
+## Import Configuration
+
+### Required configuration
+
+- `seedCluster`contains the kubeconfig for the cluster
+   - into which the Gardenlet is deployed by the landscaper
+   - that is targeted as the Seed cluster by the Gardenlet via the default in-cluster mounted service account token
+   
+   Hence, the Gardenlet is always deployed into the Seed cluster itself.
```

Comment:
```suggestion
```

Reply:
Why should that be deleted? The Gardenlet could also be deployed into a third cluster (runtime cluster) we we deliberately narrow the options here and always deploy it into the Seed cluster.

Reply:
I have no strong opinion, but when I read it last week then I had the impression that this sentence doesn't help me or rather irritates me as a reader.